lib/suspenders/adapters/heroku.rb

```diff
@@ -88,6 +88,16 @@ def set_heroku_serve_static_files
         end
       end
 
+      def set_heroku_application_host
+        heroku_app_name = app_builder.app_name.dasherize
```

Comment:
`heroku_app_name = app_builder.app_name.dasherize` is repeated in line 68. Can we extract a method from both?